packages/react-instantsearch-core/src/connectors/__tests__/connectConfigure.js

```diff
@@ -5,39 +5,65 @@ jest.mock('../../core/createConnector', () => x => x);
 
 describe('connectConfigure', () => {
   describe('single index', () => {
-    const context = { context: { ais: { mainTargetedIndex: 'index' } } };
-    const getSearchParameters = connect.getSearchParameters.bind(context);
-    const transitionState = connect.transitionState.bind(context);
-    const cleanUp = connect.cleanUp.bind(context);
+    const contextValue = { ais: { mainTargetedIndex: 'index' } };
```

Comment:
The `contextValue` seems to not be correct in that case, it should be without `ais`.